### PR TITLE
Fixing Snapcraft

### PR DESCRIPTION
**Pull request by theshadowx**
Using QSlite Database.
Using Host Theme.

 #257 #185

### CODE REVIEW

**Comment by theshadowx** [comment]
I will fix the issue concerning travis

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+name: notes
+# Note on version: $UPSTREAM_VERSION~.git$GIT_COMMIT_ID
+summary: Note taking application, write down your thoughts.
+description: |
+  Notes is a tool for the things you need to write down off your
+  brain. It is your place of expressing yourself. Notes is an open
+  source and cross-platform project, and you're welcome to be part
+  of it and contribute. We hope you will love it as much as we do.
+version: latest
+version-script: GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD);
```
Comment (bjorn) [inline snap/snapcraft.yaml]: Note that `version-script` is [deprecated](https://snapcraft.io/docs/deprecation-notices/dn10), so while you're updating the `snapcraft.yml` file you may want to use the new method for setting the version.

For Tiled I ended up with:

https://github.com/bjorn/tiled/blob/9cc4bcd6322cd2d7b73a2de1f8a594749bd29837/snap/snapcraft.yaml#L37-L39

Comment (theshadowx) [reply snap/snapcraft.yaml]: I wasn t aware of that, your repo is one of my references ;) I'll update that.

Comment (bjorn) [reply snap/snapcraft.yaml]: Hehe, though in general you have to be a bit careful using it as reference. Most of the snap packaging stuff is just guesswork or copy/paste... Please let me know if you find anything that could be improved. :-)

**Comment by bjorn** [comment]
> I will fix the issue related to travis

It seems the Travis build may be broken because of the version change in `Notes.pro`. Since it is anyway unrelated to getting the snap to build I would suggest to take that change out of this pull request and do it separately. There are a lot more places that will need to be updated regarding the version (see bb64fe95d820b5d07379d5b2fc238c4030b57118).

**Comment by theshadowx** [comment]
@bjorn sure, I'm reverting it to 1.0.0 to be updated later.

**Comment by nuttyartist** [comment]
@theshadowx Feel free to merge it when it's ready.

**Comment by nuttyartist** [comment]
@bjorn So we can change the version after this PR?

**Comment by bjorn** [comment]
> @bjorn So we can change the version after this PR?

Sure, let's open another PR for updating the version and fix any issues this causes there.

**Pull request merged**